src/tables.jl

```diff
@@ -38,10 +38,11 @@ function Tables.columns(A::Union{KeyedArray, NdaKa})
     L = hasnames(A) ? (dimnames(A)..., :value) :
         (ntuple(d -> Symbol(:dim_,d), ndims(A))..., :value)
     R = keys_or_axes(A)
-    G = ntuple(ndims(A)) do d
-        vec([rs[d] for rs in Iterators.product(R...)])
-        # _vec(rs[d] for rs in Iterators.product(R...))
-    end
+    R_inds = map(eachindex, R)
+    G = map(
```

Comment:
Can we add a comment saying what this done?
It is pretty unclear to me

Reply:
Added a comment, let me know if it makes things any clearer.